verilog/analysis/checkers/create_object_name_match_rule.cc

```diff
@@ -121,6 +121,11 @@ static absl::string_view StripOuterQuotes(absl::string_view text) {
 static const TokenInfo* ExtractStringLiteralToken(
     const SyntaxTreeNode& expr_node) {
   if (!expr_node.MatchesTag(NodeEnum::kExpression)) return nullptr;
+
+  if (expr_node.children().front().get()->Kind() != verible::SymbolKind::kLeaf) {
```

Comment:
// this check is limited to only checking string literal leaf tokens

Reply:
Sure, added